src/AllAssignmnets/Assignment1/Assignment1.java

```diff
@@ -0,0 +1,32 @@
+package AllAssignmnets.Assignment1;
+import java.io.File;
+import java.util.Scanner;
+
+public class Assignment1 {
+    public static void main(String[] args) {
+        Scanner sc=new Scanner(System.in);
+        while(true){
+            System.out.println("Enter Regular Expression: ");
+            String regularExp = sc.nextLine();
+            Assignment1 assignment = new Assignment1();
+            assignment.fileSearch("/home/ramaa/IdeaProjects", regularExp);
+        }
+    }
+    public void fileSearch(String filePath,String regularExpression) throws  NullPointerException{
```

Comment:
Please follow SOLID principles. This is voilating the single responsibility principle.

Reply:
solved